website/content/docs/nia/cli/cli-overview.mdx

```diff
@@ -46,10 +48,39 @@ Behavior: This has similar behavior as `-inspect` mode for the selected task. Th
 
 Usage: Useful to debug one or more tasks to confirm configuration is accurate and the selected tasks would update network infrastructure as expected.
 
-### Once Mode
+#### Once Mode
 
 Flag: `-once`
 
 Behavior: Consul-Terraform-Sync will run all tasks once with buffer periods disabled and exit. On encountering an error before completing, Consul-Terraform-Sync will exit with a non-zero status.
 
 Usage: Intended to be run before daemon-mode in order to confirm configuration is accurate and tasks update network infrastructure as expected.
+
+## Subcommands
+
+In addition to running the main process, Consul-Terraform-Sync features a set of subcommands to provide a user-friendly experience interacting with the main process. This functionality uses the APIs but does not correspond one-to-one with it. Please see the individual subcommands in the left navigation for more details.
```

Comment:
Whatever terminology is decided upon for the above case, it should be used here as well. The use here does lend itself to the client/server terminology as well.

Reply:
Agree! Thanks for pointing this out.